_test/add_common_test_functions_to_path.m

```diff
@@ -0,0 +1,5 @@
+function common_funcs_path = add_common_test_functions_to_path()
```

Comment:
the repeated `add_common_test_data_to_path` block should be extracted as a function in the same way.